pkgs/applications/networking/remote/aws-workspaces/default.nix

```
@@ -70,6 +74,6 @@ stdenv.mkDerivation rec {
     sourceProvenance = with sourceTypes; [ binaryNativeCode ];
     license = licenses.unfree;
     platforms = [ "x86_64-linux" ]; # TODO Mac support
-    maintainers = [ maintainers.mausch ];
+    maintainers = with maintainers; [ mausch dylanmtaylor ]
```

```suggestion
    maintainers = with maintainers; [ mausch dylanmtaylor ];
```

Missed that semicolon before :man_facepalming:

---

No worries at all, I'll add it.

---

I am going to force push though so this is a clean one-commit PR.